dali/test/python/test_operator_crop_mirror_normalize.py

```diff
@@ -457,3 +457,104 @@ def test_cmn_crop_sequence_length():
                         yield check_cmn_crop_sequence_length, device, batch_size, dtype, \
                             input_layout, input_shape, output_layout, mirror_probability, \
                             mean, std, should_pad
+
+def check_cmn_with_out_of_bounds_policy_support(device, batch_size, dtype, input_layout, input_shape, output_layout, 
+                                                mirror_probability, mean, std, should_pad,
+                                                out_of_bounds_policy=None, fill_values=(0x76, 0xb9, 0x00)):
+    # This test case is written with HWC layout in mind and "HW" axes in slice arguments
+    assert(input_layout == "HWC")
+    assert(len(input_shape) == 3)
+    if fill_values is not None and len(fill_values) > 1:
+        assert(input_shape[2] == len(fill_values))
+
+    eii = RandomDataIterator(batch_size, shape=input_shape)
+    crop_y = 0.5
+    crop_x = 0.5
+    crop_h = input_shape[0] * 2
+    crop_w = input_shape[1] * 2
+
+    pipe = CMNRandomDataPipeline(device, batch_size, input_layout, iter(eii),
+                                 dtype = dtype, output_layout = output_layout,
+                                 mirror_probability = mirror_probability, mean = mean, std = std, pad_output = should_pad,
+                                 crop_w = crop_w, crop_h = crop_h,
+                                 crop_pos_x = crop_x, crop_pos_y = crop_y,
+                                 out_of_bounds_policy = out_of_bounds_policy, fill_values = fill_values, extra_outputs = True)
+
+    permute = None
+    if output_layout != input_layout:
+        permute = []
+        for d in range(len(input_layout)):
+            perm_d = input_layout.find(output_layout[d])
+            permute.append(perm_d)
+
+    if fill_values is None:
+        fill_values = 0
+    pipe.build()
+    for k in range(3):
+        outs = pipe.run()
+        out = outs[0]
+        in_data = outs[1]
+        mirror_data = outs[2]
+        if isinstance(out, dali.backend_impl.TensorListGPU):
+            out = out.as_cpu()
+        if isinstance(in_data, dali.backend_impl.TensorListGPU):
+            in_data = in_data.as_cpu()
+
+        assert(batch_size == len(out))
+        for idx in range(batch_size):
+            sample_in = in_data.at(idx)
```

Comment:
```suggestion
            sample_in = in_data[idx]
```
and in other lines as well?